wavefront/resource_alert_test.go

```diff
@@ -168,6 +170,160 @@ func TestAccWavefrontAlert_Multiple(t *testing.T) {
 	})
 }
 
+func TestAccWavefrontAlert_Threshold(t *testing.T) {
+	var record wavefront.Alert
+	var tmpTarget *wavefront.Target
+	var err error
+	tmpTargetID := ""
+
+	if os.Getenv("TF_ACC") == "true" {
+		tmpTarget, err = createTarget()
```

Comment:
Is it possible to also create the tmp target within testAccCheckWavefrontAlert_threshold ? Then we don't need to worry about this check, creating it and tidying it up.

Thinking it the targetID can be retrieved from state within testAccCheckWavefrontThresholdAlertAttributes()